app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,54 @@
+class RecipesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @recipes = current_user.recipes.includes(:user)
+  end
+
+  def show
+    @recipe = Recipe.includes(:user, :food_name).find(params[:id])
+  end
```

Comment:
- [ ] Well! I can see the recipe show page without any error. 👍 But it seems there's a security concern in your current implementation. As per [the project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=If%20the%20recipe%20is%20public%20or%20the%20user%20is%20the%20owner%20of%20the%20recipe%2C%20should%20display%20the%20recipe%20details%20as%20in%20the%20wireframe.), users should only be able to view the public recipes of other users. To address this issue, consider updating your code to enforce the privacy settings. Ensure that the application checks whether a recipe is public before allowing users to view it.


By implementing these changes, you'll enhance the security of your application and align it with the specified project requirements. Keep up the good work! 🚀🔐